data/assets/scene/digitaluniverse/alternatestarlabels.asset

```diff
@@ -49,10 +43,10 @@ asset.export(Object)
 
 
 asset.meta = {
-  Name = "Stars Labels - Alternate",
-  Version = "2.0",
-  Description = "Digital Universe asset for alternate start labels",
-  Author = "Brian Abbott (AMNH)",
+  Name = "Alternative Labels for the Stars",
+  Version = "2.1",
+  Description = "Alternate star labels for the stars. Priority goes to Bayer IDs (Greek designations, like Alpha Orionis), then to Flamsteed numbers (like 1 Orionis).",
```

Comment:
Here's an example where the description could be re-used from the scene graph node above. This is how to do it: 
```suggestion
  Description = Object.GUI.Description,
```